crates/bevy_winit/src/system.rs

```diff
@@ -174,7 +174,7 @@ pub(crate) fn changed_windows(
                     window.resolution.physical_width(),
                     window.resolution.physical_height(),
                 );
-                winit_window.set_inner_size(physical_size);
+                let _ = winit_window.request_inner_size(physical_size);
```

Comment:
[(link to docs)](https://docs.rs/winit/0.29.3/winit/window/struct.Window.html#method.request_inner_size)
" **may** not be generated " 🙁 ; I don't like uncertainty...

this seems to [only affect wayland?](https://github.com/rust-windowing/winit/blob/21701a33de20775fd1fb11ca75b23bf5438a7acf/src/platform_impl/linux/wayland/window/mod.rs#L315-L320) ; but the exact path to the **maybe not** generated event is quite hard to follow 🤔...

I'll follow the trail to winit and might open an issue because I'd like to handle differently:
- Some(newValue): generate an event resize [like we do when reacting from events](https://github.com/bevyengine/bevy/blob/66541a085608b60da4c24bdda1861c9904fad375/crates/bevy_winit/src/lib.rs#L477C38-L486) ⚠️ the "**may**" from the docs may cause the event to be sent twice.
- Some(oldValue): do not generate an event resize ⚠️ I cannot differentiate it from a new value, so I'd send "useless" resize. Well I guess we're not supposed to send a resize event in those platforms anyway but still.
- None: event will be sent afterward, or not ✅ 

maybe the docs is wrong and should read "**will** not be generated", so I can handle it with certainty 🤞.

Reply:
Thanks for bringing up resizing, I think that PR has a small regression, as window is not redrawn during resizing.

<details><summary>My PR</summary>
<p>

https://github.com/bevyengine/bevy/assets/2290685/0971969a-cbba-4ad4-a40a-654f322bf3b4

</p>
</details> 

<details><summary>On main</summary>
<p>

(rev 0cc11791b9a55f5d310ee754c9f7254081e7138a)

https://github.com/bevyengine/bevy/assets/2290685/e7cc36b5-c0c8-4797-bc1b-a8e4bbcab938

</p>
</details>